### PR TITLE
Update description of R and ruODK in OData modal

### PR DESCRIPTION
**Pull request by florianm**
* Update the description of the statistical programming language R using words from their own homepage. 
* Clarify that ruODK supports both OData and RESTful API endpoints.

I hope I got the text formatting right!

---
This change is [Reviewable](https://reviewable.io/reviews/opendatakit/central-frontend/247)

### CODE REVIEW

File: src/components/submission/analyze.vue [hunk 1]
```diff
@@ -64,17 +64,15 @@ except according to the terms contained in the LICENSE file.
         </p>
         <template v-else-if="tool === 'r'">
           <p>
-            To access Central data from the free and popular
+            To access Central data from the free software environment for statistical computing and graphics
```
[the remark this answers is not in the document]
Comment (issa-tseng) [reply src/components/submission/analyze.vue]: i ran the existing text by @yanokwa so i'm reluctant to change it. i understand that R describes themselves some particular way but i think it's honestly an off-putting description to non-technical folks. what's a software _environment_? i have to do "computing"?

File: src/components/submission/analyze.vue [hunk 2]
```diff
@@ -64,17 +64,15 @@ except according to the terms contained in the LICENSE file.
...
-            <a href="https://dbca-wa.github.io/ruODK/" target="_blank"
-              rel="noopener">
-              ruODK</a>.
-            A guide for getting started with it can be found
-            <a href="https://dbca-wa.github.io/ruODK/articles/odata.html"
-              target="_blank" rel="noopener">
-              here</a>.
+            R</a>, we recommend you use
+            <a href="https://dbca-wa.github.io/ruODK/" target="_blank" rel="noopener">ruODK</a>.
+            See ruODK's vignettes for examples of using both the
```
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: I'll delete the trailing white space here.

File: src/components/submission/analyze.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: Agree, let's split at 80!

File: src/components/submission/analyze.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: OK, I'll replace `'` with `&rsquo;`.

File: src/components/submission/analyze.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: That was my biggest driver - R is never called a "tool" in the R community. To reduce friction, I just used the words from the R homepage itself (assuming they'd know best how to describe R). So this text change is the core of my PR.

File: src/components/submission/analyze.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: Test are still failing over the now missing previous R description. Whitespace is gone, resolving this comment.

File: src/components/submission/analyze.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: Split at 80-ish, resolving.

File: src/components/submission/analyze.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: Done, resolving.

File: src/components/submission/analyze.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (issa-tseng) [reply src/components/submission/analyze.vue]: a lot of people have no idea what R is, so a little friendly description is certainly not a bad thing. if it's okay with you, i'd like to leave that part of the sentence alone.

[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: Friendly and not technical/intimidating is good.
Calling R a "tool" on the other hand scratches the eyeballs of R users - finding better but still correct wording here is the sole reason for this PR.

Options:

- Drop explanation of R, see PowerBI (if you use it you know what it is) - as you rightly say, that's not friendly for users which are new to R.
- Use wording from official R page - a but technical, true that. At least no one from the R community can argue with R's preferred self-description.
- Use something not too technical but also not obviously wrong. How about "the statistical programming language R"?

[the remark this answers is not in the document]
Comment (issa-tseng) [reply src/components/submission/analyze.vue]: yes, i'd rather drive people toward R than toward Power BI, so i am not trying to do the same thing in both texts. calling it a programming language will certainly not accomplish that, however.

[the remark this answers is not in the document]
Comment (issa-tseng) [reply src/components/submission/analyze.vue]: i think i'm going to just let @yanokwa make a decision here.

[the remark this answers is not in the document]
Comment (florianm) [reply src/components/submission/analyze.vue]: Happy to go with your preference. That's taken already more of your brain time than I want to occupy with this PR ;-)

**Comment by yanokwa** [comment]
I'm happy the PR as it stands. @matthew-white what's blocking the merge?

**Comment by issa-tseng** [comment]
only my whining.

**Pull request merged**